person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permision: true)
+    @id = Random.rand(1..10)
+    @name = name
+    @age = age
+    @parent_permission = parent_permision
+  end
+
+  def can_use_services?
+    if is_of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
+
+  attr_reader :id
+  attr_accessor :name, age
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you move all your attributes(`attr_reader` and `attr_accessor`) just the `iniatialiaze` constructor method and make sure you correct the typing error on the `attr_accessor: name, age` to be `attr_accessor: name, :age`